src/CosmosEditorManager.ts

```diff
@@ -21,153 +22,153 @@ import { MongoDocumentTreeItem } from './mongo/tree/MongoDocumentTreeItem';
 import * as vscodeUtils from './utils/vscodeUtils';
 
 export interface ICosmosEditor<T = {}> {
-    label: string;
```

Comment:
Can you revert all of these whitespace changes? Perhaps you have a VS Code setting that is auto-formatting?

Also please fix the merge conflicts in this file (might be related to your whitespace changes)

Reply:
Is there an [editorconfig](https://EditorConfig.org) file that describes the whitespace preferences for the project? Perhaps add one to the repo?